features/us65_delete_borrower_personal_info.feature

```diff
@@ -0,0 +1,19 @@
+@us65_delete_borrower_personal_info
+
+Feature: Delete borrower personal information once the deed has been submitted
+  In order to ensure that Land Registry is not holding onto personal information
+  Borrowers personal details must be deleted from Land Registry records once the
+  mortgage application has been registered.
+
+Background:
+  Given I have valid deed data
+  And I create the deed via the Deed API
+  And I know the borrower id
+
+Scenario: Delete Single Borrowers Details
+  When I delete the deed via the Deed API
+  Then a status code of "200" is returned
```

Comment:
It hasnt been covered in unit test, Simon T thought it should be an integration test but Ramin didn't think it was possible. So have added an extra step to try to delete the personal info again with an expected 404 response as it no longer exists